dubbo-common/src/main/java/org/apache/dubbo/rpc/model/ConsumerModel.java

```diff
@@ -18,6 +18,7 @@
 
 import org.apache.dubbo.common.utils.Assert;
 import org.apache.dubbo.config.ReferenceConfigBase;
+import org.apache.dubbo.config.annotation.Service;
```

Comment:
Please delete unused import, otherwise you can't pass ci check.

Reply:
thanks, will do.